hack/verify-flags/known-flags.txt

```diff
@@ -10,10 +10,10 @@ algorithm-provider
 all-namespaces
 allocate-node-cidrs
 allow-privileged
+api-server-address
```

Comment:
nvm. It should be useful to most kube users. So its ok to be here. 
I just find it weird that we group test-only and production flags in a single file. 

This file should probably be per-component.